src/util/runtime_configuration.cpp

```diff
@@ -133,7 +133,7 @@ namespace hpx { namespace util
             "[hpx]",
             "location = ${HPX_LOCATION:$[system.prefix]}",
             "component_paths = ${HPX_COMPONENT_PATHS}",
-            "component_base_paths = $[hpx.location]"
+            "component_base_paths = $[hpx.location]" // NOLINT
```

Comment:
What warning did this cause?

Reply:
use after move, which this code is explicitly testing for.

Reply:
The one in `tests/unit/util/any.cpp` I get, but this? No test here...

Reply:
grr, sorry. actually readin where you commented helps ;)
This was:
`/home/heller/programming/hpx/src/util/runtime_configuration.cpp:136:13: error: suspicious string literal, probably missing a comma [bugprone-suspicious-missing-comma,-warnings-as-errors]`